endoreg_db/serializers/Frames_NICE_and_PARIS_classifications.py

```
@@ -0,0 +1,498 @@
+
+        segment_predictions.sort(key=lambda x: x["low_quality"])
+
+        selected_frames = []
+        last_selected_frame = -float('inf')
+
+        for frame in segment_predictions:
+            if len(selected_frames) >= FRAMES_PER_SEQUENCE:
+                break
+
+            if all(rule(frame["prediction"]) for rule in FRAME_SELECTION_RULES):
+                if frame["frame_number"] - last_selected_frame >= MIN_FRAME_GAP_IN_SEQUENCE:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)